test/scraper/schleswigholstein_landtag_scraper_overview_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'test_helper'
+
+class SchleswigholsteinLandtagScraperOverviewTest < ActiveSupport::TestCase
+  def setup
+    @scraper = SchleswigHolsteinLandtagScraper
+    @html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_overview.html')
+                           ).force_encoding("WINDOWS-1252"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.